main.js

```diff
@@ -162,10 +162,134 @@ const inventory = [
     },
 ];
 
-function calculateAmountToSell(array) {
+
+function calculateAmountToSell(inventoryArray) {
     let stockToSell = 0;
     for (const inventoryKey in inventory) {
-        stockToSell += inventoryKey.originalStock - inventoryKey.sold;
+        stockToSell += inventory[inventoryKey].originalStock - inventory[inventoryKey].sold;
     }
     return stockToSell;
-}
\ No newline at end of file
+}
+
+function calculateRevenueTarget(inventoryArray) {
+    let revenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        revenue += (inventoryArray[inventoryKey].originalStock - inventoryArray[inventoryKey].sold) * inventoryArray[inventoryKey].price;
+    }
+    return revenue;
+}
+
+function calculateCurrentRevenue(inventoryArray) {
+    let currentRevenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        currentRevenue += inventoryArray[inventoryKey].sold * inventoryArray[inventoryKey].price;
+    }
+    return currentRevenue;
+}
+
+const revenueTarget = document.getElementById("revenueTarget");
+revenueTarget.textContent += calculateRevenueTarget(inventory);
+revenueTarget.style.color = "blue";
```

Comment:
Als je geen dynamische css hebt, dan zie ik deze het liefst in de daarvoor bestemde `.css` files